modules/services/barrier.nix

```diff
@@ -0,0 +1,75 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let cfg = config.services.barrier;
+in {
+
+  meta.maintainers = with maintainers; [ kritnich ];
+
+  options.services.barrier = {
+
+    client = {
+
+      enable = mkEnableOption "Barrier Client daemon";
+
+      name = mkOption {
+        type = types.nullOr types.str;
+        default = null;
+        description = ''
+          Screen name of client. Defaults to hostname.
+        '';
+      };
+
+      server = mkOption {
+        type = types.str;
+        default = null;
+        description = ''
+          Server to connect to formatted as
+          <literal>&lt;host&gt;[:&lt;port&gt;]</literal>.
+          Port defaults to 24800. Required.
+        '';
+      };
+
+      tray = mkEnableOption "the system tray icon" // { default = true; };
+
+      enableCrypto = mkEnableOption "crypto (SSL) plugin" // {
+        default = true;
+      };
+
+      enableDragDrop = mkEnableOption "file drag &amp; drop";
+
+      extraFlags = mkOption {
+        type = types.listOf types.str;
+        default = [ "-f" ];
```

Comment:
```suggestion
        default = [ "-f" ];
        defaultText = literalExample ''[ "-f" ]'';
```

Reply:
Not very important but it should be just fine without the `defaultText`. Looks like
![without-literal](https://user-images.githubusercontent.com/798147/116756421-5a861c00-aa0c-11eb-8b2e-0d04a35c9ab4.png)